docs/reference/env-vars.md

```diff
@@ -92,6 +92,11 @@ The following environment variables control the configuration of the Nextflow ru
   :::
 : Enable Nextflow *strict* execution mode (default: `false`)
 
+`NXF_ENABLE_STRICT_SYNTAX`
```

Comment:
```suggestion
`NXF_ENABLE_SYNTAX_PARSER_V2` (?)
```

Reply:
Well, I think it will be better to call it `NXF_ENABLE_STRICT_SYNTAX` to keep consistent with our terminology around [strict syntax](https://nextflow.io/docs/latest/updating-syntax.html#preparing-for-strict-syntax). It's not just a new and improved parser, it's a formal language spec.

Reply:
The strict feature flag won't be needed as we continue with the strict language spec, so I would rather just phase it out than try to work around it.